Formula/skaa.rb

```diff
@@ -0,0 +1,44 @@
+class Skaa < Formula
+  desc "Seven Kingdoms: Ancient Adversaries"
+  homepage "https://7kfans.com"
+  url "https://github.com/the3dfxdude/7kaa/releases/download/v2.15.5/7kaa-2.15.5.tar.xz"
+  sha256 "350a2681985feb4b71d20677d1a6d11921b9e3d97facbc94e4f020e848d8ab2b"
+  license "GPL-2.0-only"
+
+  # we only use gcc for building the app
+  depends_on "gcc@11" => :build
+  depends_on "enet"
+  depends_on "gettext"
+  depends_on "sdl2"
+
+  uses_from_macos "curl"
+
+  def install
+    # ensure we use gcc 11 for building the app, Clang will fail
+    ENV["CXX"] = "/usr/local/Cellar/gcc@11/11.3.0/bin/g++-11"
+
+    # configure the build
+    system "./configure", "--prefix=#{prefix}"
+
+    # build the app
+    system "make"
+    system "make", "install"
+  end
+
+  def caveats
+    <<~EOS
+      The start the game, go to the SKAA install directory (#{prefix}) and type:
+        7kaa
+
+      In order to easily start the game outside the install directory, type: brew link skaa, afterwards you can run the game anywhere by typing:
+        7kaa
+
+      The music bundle is not GPL but can be used freely with Seven Kingdoms: Ancient Adversaries. Download the music and move to data folder. The music can be downloaded from:
+        https://www.7kfans.com/downloads/7kaa-music-2.15.zip
+    EOS
+  end
```

Comment:
```suggestion
  end
```
These are not suitable for `caveats`.

Reply:
I guess the caveat regarding music is already in the README. However, I think it can be confusing for people that the formula name is skaa while the binary itself is called 7kaa. However, it seems I cannot make a formula that starts with a number. I wish it'd be possible to rename the binary into skaa using the formula, but I don't think it's possible.